exchange/adapter_builders.go

```diff
@@ -196,6 +196,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderTelaria:          telaria.Builder,
 		openrtb_ext.BidderTriplelift:       triplelift.Builder,
 		openrtb_ext.BidderTripleliftNative: triplelift_native.Builder,
+		openrtb_ext.BidderTrustx:           grid.Builder,
```

Comment:
The title of this PR is using the capitalization "TrustX". If that's the proper branding, please feel free to use that in the bidder name and user sync name. `BidderTrustX` and `NewTrustXSyncer` respectively.

Reply:
Thank you, renamed.